### PR TITLE
5.4.x testnet node release

### PR DESCRIPTION
**Pull request by dg-concordium**
## Purpose

Inform users about testnet node release.

## Changes

Release notes, testnet downloads, previous nodes

## Checklist

- [X] My code follows the style of this project.
- [X] The code compiles without warnings.
- [X] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

_Remove if not applicable.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [X] I accept the above linked CLA.

### CODE REVIEW

**Comment by dg-concordium** [comment]
This also contains the release for the client which will be for both testnet and mainnet, hence mainnet release notes.

File: source/mainnet/net/installation/downloads-testnet.rst [hunk 1]
```diff
@@ -152,41 +152,41 @@ For the system requirements to run a node, see :ref:`System requirements to run
...
 
-Windows v5.3.2
---------------
+Windows |testnet-node-version|
+------------------------------
 
 To run a node on Windows, you need a Windows Installer package. **Please be aware that you should backup your configuration, as the installer will overwrite the current configuration with a standard configuration.**
 
-   - `Download the Testnet Windows Installer package <https://distribution.concordium.software/windows/Signed/Node-5.3.2-0.msi>`_
+   - `Download the Testnet Windows Installer package <https://distribution.concordium.software/windows/Signed/Node-5.4.0-0.msi>`_
```
Comment (mh-concordium) [inline source/mainnet/net/installation/downloads-testnet.rst:180]: There is as new package for all the  distributions, we will release 5.4.1.

File: source/mainnet/net/installation/downloads-testnet.rst [hunk 2]
```diff
@@ -152,41 +152,41 @@ For the system requirements to run a node, see :ref:`System requirements to run
...
+Ubuntu |testnet-node-version|
+-----------------------------
 
 To run a node on a server with Ubuntu, you need a Debian package.
 
-   - `Download the Testnet Debian package <https://distribution.testnet.concordium.com/deb/concordium-testnet-node_5.3.2-0_amd64.deb>`_
+   - `Download the Testnet Debian package <https://distribution.testnet.concordium.com/deb/concordium-testnet-node_5.4.0-0_amd64.deb>`_
 
-      - SHA256 checksum of the download: ``cf035d33784473b13afdf6d9a047c35f970ad9864f1134867eed00ed7baa2007``
+      - SHA256 checksum of the download: ``62c24f7a2bc48ba31a7143f65d16f14b282b46940ebf093b1ffaaa258ba1a9da``
```
Comment (mh-concordium) [inline source/mainnet/net/installation/downloads-testnet.rst:162]:
```suggestion
      - SHA256 checksum of the download: ``8c1f8df5c83999c10a4cda8f9d35f632f38c23aa797a50fcb4db3c591788b291``
```

File: source/mainnet/net/installation/downloads-testnet.rst [hunk 3]
```diff
@@ -152,41 +152,41 @@ For the system requirements to run a node, see :ref:`System requirements to run
...
 
-Ubuntu v5.3.2
--------------
+Ubuntu |testnet-node-version|
+-----------------------------
 
 To run a node on a server with Ubuntu, you need a Debian package.
 
-   - `Download the Testnet Debian package <https://distribution.testnet.concordium.com/deb/concordium-testnet-node_5.3.2-0_amd64.deb>`_
+   - `Download the Testnet Debian package <https://distribution.testnet.concordium.com/deb/concordium-testnet-node_5.4.0-0_amd64.deb>`_
```
Comment (mh-concordium) [inline source/mainnet/net/installation/downloads-testnet.rst:160]:
```suggestion
   - `Download the Testnet Debian package <https://distribution.testnet.concordium.com/deb/concordium-testnet-node_5.4.1-0_amd64.deb>`_
```

File: source/mainnet/net/installation/downloads-testnet.rst (hunk 1 above)
Comment (mh-concordium) [inline source/mainnet/net/installation/downloads-testnet.rst:180]:
```suggestion
   - `Download the Testnet Windows Installer package <https://distribution.concordium.software/windows/Signed/Node-5.4.1-0.msi>`_
```

File: source/mainnet/net/installation/downloads-testnet.rst [hunk 4]
```diff
@@ -152,41 +152,41 @@ For the system requirements to run a node, see :ref:`System requirements to run
...
 
-Mac v5.3.2
-----------
+Mac |testnet-node-version|
+--------------------------
 
 To run a node on macOS, you need a macOS installer package.
 
-   - `Download the Testnet macOS installer package <https://distribution.concordium.software/macos/signed/concordium-node-5.3.2.pkg>`_
+   - `Download the Testnet macOS installer package <https://distribution.concordium.software/macos/signed/concordium-node-5.4.0.pkg>`_
```
Comment (mh-concordium) [inline source/mainnet/net/installation/downloads-testnet.rst:189]:
```suggestion
   - `Download the Testnet macOS installer package <https://distribution.concordium.software/macos/signed/concordium-node-5.4.1.pkg>`_
```

**Review by mh-concordium — CHANGES_REQUESTED** [review]

File: source/mainnet/net/resources/release-notes-mainnet.rst [hunk 5]
```diff
@@ -11,6 +11,17 @@ Release notes - Mainnet
...
    To subscribe to updates on the Mainnet status page click **Subscribe** to get all updates or click **Get updates** to choose to get all updates or only updates for specific products.
 
+June 1, 2023
+
+Concordium Client 5.2.0
+-----------------------
+
+Version 5.2.0 contains the following features and bug fixes:
+
+- Fixed a bug that caused another code-leftover to be displayed in the finalization proof line of the gas rewards section of the ``consensus show-chain-parameters`` output. This is only relevant when the queried chain runs protocol version 6.
```
Comment (abizjak) [inline source/mainnet/net/resources/release-notes-mainnet.rst:21]:
```suggestion
- Fix a bug in display of ``consensus show-chain-parameters`` output for protocol version 6.
```

File: source/mainnet/net/resources/release-notes.rst [hunk 6]
```diff
@@ -11,6 +11,32 @@ Release Notes - Testnet
...
+- Extend gRPC v2 API call ``GetBlockInfo`` with the protocol version of the block.
+
+- Do not use peer buckets when running as a normal node (not as a bootstrapper).
+
+Concordium Client 5.2.0
+-----------------------
+
+Version 5.2.0 contains the following features and bug fixes:
+
+- Fixed a bug that caused another code-leftover to be displayed in the finalization proof line of the gas rewards section of the ``consensus show-chain-parameters`` output. This is only relevant when the queried chain runs protocol version 6.
```
Comment (abizjak) [inline source/mainnet/net/resources/release-notes.rst:36]:
```suggestion
- Fix a bug in display of ``consensus show-chain-parameters`` output for protocol version 6.
```

File: source/mainnet/net/resources/release-notes.rst [hunk 7]
```diff
@@ -11,6 +11,32 @@ Release Notes - Testnet
...
+
+- Added ``raw GetBlockTransactionEvents`` that prints the list of transaction outcomes in a given block.
```
Comment (abizjak) [inline source/mainnet/net/resources/release-notes.rst:38]:
```suggestion
- Add ``raw GetBlockTransactionEvents`` that prints the list of transaction outcomes in a given block.
```

File: source/mainnet/net/resources/release-notes-mainnet.rst [hunk 8]
```diff
@@ -11,6 +11,17 @@ Release notes - Mainnet
...
+
+- Added ``raw GetBlockTransactionEvents`` that prints the list of transaction outcomes in a given block.
```
Comment (abizjak) [inline source/mainnet/net/resources/release-notes-mainnet.rst:23]:
```suggestion
- Add ``raw GetBlockTransactionEvents`` that prints the list of transaction outcomes in a given block.
```

File: source/mainnet/net/resources/release-notes.rst [hunk 9]
```diff
@@ -11,6 +11,32 @@ Release Notes - Testnet
...
+
+- Enable CORS support in grpc-web. This only applies when grpc-web is enabled.
+
+- Fixed a security issue.
+
+- Support using block height as block identifiers in gRPC v2 API.
+
+- Extend gRPC v2 API call ``GetBlockInfo`` with the protocol version of the block.
+
+- Do not use peer buckets when running as a normal node (not as a bootstrapper).
```
Comment (abizjak) [inline source/mainnet/net/resources/release-notes.rst:29]:
```suggestion
- Do not keep a historical list of peers when running as a normal node.
```

**Review by abizjak — APPROVED** [review]
I have some suggestions for making the changelogs a bit nicer.

Looks good otherwise.

**Pull request merged**